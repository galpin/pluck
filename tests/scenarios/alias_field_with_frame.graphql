{
  cores(limit: 10) {
    mission: missions @frame {
      name
      flight
    }
  }
}