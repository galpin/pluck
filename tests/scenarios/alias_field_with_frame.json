{
  "data": {
    "cores": [
      {
        "mission": [
          {
            "name": "CRS-6",
            "flight": 22
          }
        ]
      },
      {
        "mission": [
          {
            "name": "CRS-1",
            "flight": 9
          }
        ]
      },
      {
        "mission": [
          {
            "name": "Inmarsat-5 F4",
            "flight": 40
          }
        ]
      },
      {
        "mission": [
          {
            "name": "TürkmenÄlem 52°E / MonacoSAT",
            "flight": 23
          }
        ]
      },
      {
        "mission": [
          {
            "name": "CRS-9",
            "flight": 32
          },
          {
            "name": "Falcon Heavy Test Flight",
            "flight": 55
          }
        ]
      },
      {
        "mission": [
          {
            "name": "Boeing X-37B OTV-5",
            "flight": 47
          },
          {
            "name": "SES-12",
            "flight": 63
          }
        ]
      },
      {
        "mission": [
          {
            "name": "Iridium NEXT Mission 3",
            "flight": 48
          },
          {
            "name": "Iridium NEXT Mission 5",
            "flight": 58
          }
        ]
      },
      {
        "mission": [
          {
            "name": "JCSAT-2B",
            "flight": 29
          }
        ]
      },
      {
        "mission": [
          {
            "name": "DemoSat",
            "flight": 2
          }
        ]
      },
      {
        "mission": [
          {
            "name": "AsiaSat 8",
            "flight": 16
          }
        ]
      }
    ]
  }
}