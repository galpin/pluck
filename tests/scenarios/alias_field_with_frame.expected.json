{
 "mission": [
  {
   "name": "CRS-6",
   "flight": 22
  },
  {
   "name": "CRS-1",
   "flight": 9
  },
  {
   "name": "Inmarsat-5 F4",
   "flight": 40
  },
  {
   "name": "T\u00fcrkmen\u00c4lem 52\u00b0E / MonacoSAT",
   "flight": 23
  },
  {
   "name": "CRS-9",
   "flight": 32
  },
  {
   "name": "Falcon Heavy Test Flight",
   "flight": 55
  },
  {
   "name": "Boeing X-37B OTV-5",
   "flight": 47
  },
  {
   "name": "SES-12",
   "flight": 63
  },
  {
   "name": "Iridium NEXT Mission 3",
   "flight": 48
  },
  {
   "name": "Iridium NEXT Mission 5",
   "flight": 58
  },
  {
   "name": "JCSAT-2B",
   "flight": 29
  },
  {
   "name": "DemoSat",
   "flight": 2
  },
  {
   "name": "AsiaSat 8",
   "flight": 16
  }
 ]
}